app/models/organisation.rb

```diff
@@ -548,6 +548,10 @@ def public_path(options = {})
     append_url_options(base_path, options)
   end
 
+  def link_to_section_on_organisation_list_page
+    append_url_options("/government/organisations", anchor: slug)
```

Comment:
I don't think this is the first place where we've done this, but I do worry slightly that we may be duplicating hard-coded strings like "/government/organisations" around the codebase which might make it hard to change the URL path in the future...

Reply:
Good point. I think we'll leave it for this PR since there are lots of places this happens beyond this model and we want to get the fix in, but maybe one to think about if we do further url maker things